src/sdk/lsf-sdk.js

```diff
@@ -629,6 +629,18 @@ export class LSFWrapper {
     }
   };
 
+  saveDraft = () => {
+    const selected = this.lsf?.annotationStore?.selected;
+
+    const hasChanges = !!selected?.history.undoIdx;
+
+    if (!hasChanges || !selected) return;
+    selected.saveDraftImmediatelyWithResults((status) => {
+      if (status === 200 || status === 201) this.datamanager.invoke("toast", "Draft saved successfully", "info");
```

Comment:
```suggestion
      if (status === 200 || status === 201) this.datamanager.invoke("toast", { message: "Draft saved successfully", type: "info" });
```
Also it would be good to have info be the default if unspecified as to the type property. I'm thinking having a key:val object lends itself a bit to a more scalable options to this call, as its then not dependent on knowing the order. You have to just know the keys, which most toast calls usually have message and type, so its at least a guessable api.

Reply:
obviously this is semi-stylistic in nature, but even without typescript it tends to be the most common api to a toast call.